src/collections/_documentation/platforms/javascript/index.md

```diff
@@ -1022,6 +1029,109 @@ basic stack trace. This exception is stored here for further data extraction.
 
 : For breadcrumbs created from HTTP requests done via the legacy `XMLHttpRequest` API. This holds the original xhr object.
 
+## Quick Reference
+This section serves as an index/quick reference for all the config options, top level methods, etc.
+
+### Structured Contexts
```

Comment:
isnt this duplicated from earlier?

Reply:
It is. Still thinking on the best way to do the Quick Reference without it being so repetitive.

Reply:
i dont really have an opinion on whats best here, but maybe it's better to wait on building a quick ref kind of thing and get the core shipped and then add "features" if you will